src/modules/admin/search/Search.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback, useEffect } from 'react';
+import { useHistory } from 'react-router';
+
+import { Button } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+
+import csx from './Search.scss';
+
+import { Form, useQueryParams, Url, isJSONString } from 'utils';
+
+const CONFIG: Form.Config = [
+  { label: 'Query', value: '' }
+]
+
+const QUERY = 0;
+
+namespace Search {
+  export interface Props {
+    name: string;
+  }
+}
+
+const AdminSearch = (props: Search.Props): JSX.Element=> {
+  const { location, push } = useHistory();
+
+  const { name } = props;
+
+  const [query] = useQueryParams(
+    'query'
+  );
+  
+  const [{ fields }, change, directChange, submit] = Form.useManager(CONFIG);
+
+	const handleSubmit = useCallback(
```

Comment:
You using formatting on save ? because this code have different formatting

Reply:
No, I had to do that manually